docs/UserGuide.adoc

```diff
@@ -74,88 +74,100 @@ Examples:
 Shows a list of all persons in the address book. +
 Format: `list`
 
-=== Editing a person : `edit`
+=== Student Management
 
-Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+==== Enrolling a student in a class : `enroll`
+
+Enrolls a student in a class in the program. +
+
+Format: `enroll MATRIC_NUMBER CLASS_CODE MOD_CODE'
 
 ****
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index *must be a positive integer* 1, 2, 3, ...
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person's tags by typing `t/` without specifying any tags after it.
+* Enrolls the person with the given matriculation number in the class given by the class code of the module given by the module code.
 ****
 
 Examples:
 
-* `edit 1 p/91234567 e/johndoe@example.com` +
-Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-* `edit 2 n/Betsy Crower t/` +
-Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `enroll A01234567J G06 CS2101` +
+Enrolls the student with the matriculation number A01234567J in tutorial G06 of module CS2101.
 
-=== Locating persons by name: `find`
+==== Removing a student from a class : `kick`
 
-Finds persons whose names contain any of the given keywords. +
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Removes a student from a class in the program. +
+Format: `kick MATRIC_NUMBER CLASS_CODE MOD_CODE`
 
 ****
-* The search is case insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search). e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+* Removes the person with the given matriculation number from the class given by the class code of the module given by the module code.
```

Comment:
Maybe could be split into smaller dot points.
eg.
- Removes the person with the given matriculation number from the specified class
- The class should belong to the module with the MODULE_CODE
- The class should already in the program with the existing CLASS_CODE